AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithIAMIntegrationTests/README.md

```diff
@@ -0,0 +1,90 @@
+## GraphQL with IAM Auth Integration Tests
+
+The following steps show how to set up an GraphQL endpoint with AppSync. The auth configured will be IAM. This set up is used to run the tests in `GraphQLWithIAMIntegrationTests.swift`.
+
+### Set-up
+
+TODO: Figure out how to use Amplify CLI to  `amplify add api` and select `IAM` auth. Currently it does not set up correct IAM policies. 
+
+
+### Old notes
+
+// MARK: set up
+    /*
+    These instructions for `todoGraphQLWithIAM` is to set up the GraphQL appsync endpoint with
+    - Default authorization mode to be "AWS Identity and Access Management (IAM)"
+    - Cognito identity pool with no guess access
+    === WARNING: this is not working without setting IAM policy to allow GraphQL operations ===
```

Comment:
This will not render correctly. Maybe you intended this to be a quote, or a bold?

Reply:
updated